pkg/helpers/imageregistry/client.go

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package imageregistry
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
+	clusterv1 "open-cluster-management.io/api/cluster/v1"
+)
+
+const (
+	// ClusterImageRegistryLabel value is namespace.managedClusterImageRegistry
+	ClusterImageRegistryLabel = "open-cluster-management.io/image-registry"
```

Comment:
what the different between the ClusterImageRegistryLabel and ClusterImageRegistriesAnnotation?

Reply:
ClusterImageRegistryLabel is to show which managedClusterImageRegistry is used in the mangedCluster,  ClusterImageRegistriesAnnotation is the value of image registries.

Reply:
But I did not find any use of `ClusterImageRegistryLabel `.

Reply:
removed ClusterImageRegistryLabel which is not used anymore.